app/App.js

```diff
@@ -224,7 +224,7 @@ class App extends React.Component {
 
   renderFooter () {
     return (
-      <Footer className={styles.footer}>
+      <div className={styles.footer}>
```

Comment:
Better to use the footer tag here for semantics

Reply:
Done.